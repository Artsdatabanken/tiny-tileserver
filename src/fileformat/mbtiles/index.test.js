const MbTilesHandler = require("./");

const mbtiles = new MbTilesHandler();

test("get pbf", async () => {
  const cursor = {
    type: "mbtiles",
    content: { format: "pbf" },
    physicalDir: "./testdata/pbf.mbtiles",
    pathSegments: [],
    query: {},
    link: ""
  };
  const fragment = [];
  const ext = "";
  await mbtiles.load(cursor).then(value => expect(value).toMatchSnapshot());
});
