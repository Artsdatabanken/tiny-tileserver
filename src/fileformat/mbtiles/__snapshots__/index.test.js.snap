// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get pbf 1`] = `undefined`;
